### PR TITLE
21347 fix email output for sp/gp registration

### PR DESCRIPTION
**Pull request by eason-pan-bc**
*Issue #:* /bcgov/entity#21347

*Description of changes:*
- Remove: “In addition, the following links can assist you:”
- Remove: “A message from the Minister of Citizens’ Services and the Minister of Jobs, Economic Recovery and Innovation”
- Remove the bullet from the beginning of the last line in the email and move that last sentence up to the end of the paragraph that begins with ‘Important:.. ‘.
- Test it by running emailer locally and see the rendered email page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by JazzarKarim — APPROVED** [review]
LGTM! Great job 👍 Let's try to get this merged before TEST release tomorrow please.

Also, please wait for another approval before merging. Thanks!

File: queue_services/entity-emailer/src/entity_emailer/email_templates/REG-COMPLETED.html
Comment (chenhongjing) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/REG-COMPLETED.html:41]: Can you confirm with Andy if this sentence "All businesses in British Columbia...." should be in a separate paragraph as it is shown in updated design?

**Review by chenhongjing — APPROVED** [review]

**Pull request merged**
